### PR TITLE
Add reboot node case

### PR DESCRIPTION
**Pull request by sgaoshang**
[OCP-51087 - [ipi-on-nutanix] reboot worker/control nodes](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-51087)

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-nutanix-ipi-zones-single-osimage-customized-f28

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-nutanix-ipi-zones-single-osimage-customized-f28

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml [hunk 1]
```diff
@@ -324,6 +324,7 @@ tests:
     cluster_profile: nutanix-qe-zone
     env:
       CLUSTER_OS_IMAGE: https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.16-9.4/builds/416.94.202405171948-0/x86_64/rhcos-416.94.202405171948-0-nutanix.x86_64.qcow2
+      ENABLE_REBOOT_CHECK: "true"
```
Comment (jinyunma) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml:327]: suggest to append key word "reboot-nodes" on the job name to reflect the test

Comment (sgaoshang) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml:327]: Fixed, pls take a look.

**Comment by jinyunma** [comment]
/lgtm

also cc @jianlinliu @gpei to review

**Pull request merged**